whylog/assistant/__init__.py

```diff
@@ -0,0 +1,91 @@
+import abc
+
+
+class AbstractAssistant(object):
+    @abc.abstractmethod
```

Comment:
in all other instances in this project we use `from abc import abstractmethod` and then the `@abstractmethod` decorator. This way it almost looks like a part of the language. Please change it for consistency.